Week2/todo_app/program.js

```diff
@@ -2,17 +2,17 @@
 // that we can use to connect to a MySQL process
 // and access its databases.
```

Comment:
Try to commit only what's relevant for the PR you're submitting. It can be a bit confusing to have to read past all of your week2 homework to get to the week3 homework.